Sources/AccioKit/Commands/Protocols/DependencyInstaller.swift

```diff
@@ -34,9 +34,16 @@ extension DependencyInstaller {
                 let frameworkCheckoutPath: String = checkoutsDirUrl.appendingPathComponent(fileName).path
 
                 if try FileManager.default.isDirectory(atPath: frameworkCheckoutPath) {
-                    try bash("git -C '\(frameworkCheckoutPath)' reset HEAD --hard --quiet 2> /dev/null")
-                    try bash("git -C '\(frameworkCheckoutPath)' clean -fd --quiet 2> /dev/null")
-                    try bash("git -C '\(frameworkCheckoutPath)' clean -fdX --quiet 2> /dev/null")
+                    do {
```

Comment:
Why do the `git` commands fail although I added `2> /dev/null`? Is it that only the standard output is redirected via `2>` but not the error output? If possible, I would prefer a fix of the `2>` rather than a try-catch.

Reply:
I guess we could direct errors to `dev/null` by using `2>&1 /dev/null`.

However if an error occurs resetting, isn't it better to handle this by deleting the checkout so that Swift PM will redownload it? If we don't delete it and leave it as is, Swift PM will probably fail (wasn't this the reason the reset commands were implemented in the first place?).

Reply:
Yes, handling the case and deleting the checkouts is probably the better approach. I was just suggesting the alternative to save time as it would also fix the issue. But feel free to continue with a more enhanced fix. 👍